### ISSUE TITLE
Default AppPool in 3.5 image is incorrect

### ISSUE THREAD

**Issue by shirhatti** [Microsoft/aspnet-docker#4]
The default AppPool should be changed to a v2.0 AppPool

**Comment by shirhatti** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Change default AppPool for ASP.NET 3.5 images](https://trello.com/c/va86s65d/5-change-default-apppool-for-asp-net-3-5-images)

**Comment by shirhatti** [comment]
```
New-IISSite -Name test4 -PhysicalPath C:\inetpub\wwwroot\test3 -BindingInformation "*:1234:"
Start-IISCommitDelay
$a = get-iissite -Name test4
$a.Applications["/"].ApplicationPoolName = ".NET v2.0"
Stop-IISCommitDelay 
```

**Comment by rainmakerho** [comment]
hi shirhatti,
   docker build will cause error "Unknow instruction: START_IISCOMMITDELAY"

**Comment by shirhatti** [comment]
@shhsu Could you take a look?

**Comment by shhsu** [comment]
@rainmakerho 

This docker file has several problems and will definitely not work.

* Is there a reason why you are trying to install .net 3.5 on your image? Is it a .net 3.5 site?
* can you follow the instruction on the main page? You want to copy the proper docker file to your  C:\randomanswers and build your image form there.

Also, please open a new issue for this if you have further questions. Feel free to assign it to me. This issue is not for tracking general questions

**Comment by shirhatti** [comment]
Fixed by #20
